### PR TITLE
clean sonar 

### PR DESCRIPTION
**Pull request by DariuszCzajkowski**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/BookParser.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
     private static final String PATHNAME = "baza.json";
-    private static List<Book> parseJson ;
+    private List<Book> parseJson ;
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/BookParser.java]: forma czasownikowa nazwy zmiennej?

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**